source/draw/gpu/opengl/AndroidContext.ooc

```diff
@@ -60,67 +60,82 @@ AndroidContext: class extends OpenGLContext {
 		length := channels * eglImage size area
 		recover := func (b: ByteBuffer) -> Bool {
 			eglImage buffer unlock()
-			this recyclePacker(gpuRgba)
+			this _recyclePacker(gpuRgba)
 			false
 		}
 		(ByteBuffer new(sourcePointer, length, recover), promise)
 	}
-	toRaster: func ~monochrome (source: OpenGLMonochrome) -> RasterImage {
-		(buffer, promise) := this toBuffer(source, this _packMonochrome)
+	_toRaster: func ~monochrome (source: OpenGLMonochrome) -> RasterImage {
+		(buffer, promise) := this _toBuffer(source, this _packMonochrome)
 		promise free()
 		RasterMonochrome new(buffer, source size)
 	}
-	toRaster: func ~uv (source: OpenGLUv) -> RasterImage {
-		(buffer, promise) := this toBuffer(source, this _packUv)
+	_toRaster: func ~uv (source: OpenGLUv) -> RasterImage {
+		(buffer, promise) := this _toBuffer(source, this _packUv)
 		promise free()
 		RasterUv new(buffer, source size)
 	}
 	toRaster: override func (source: GpuImage) -> RasterImage {
-		match (source) {
-			case (image : OpenGLMonochrome) =>
-				this isAligned(image channels * image size x) ? this toRaster(image) : super(image)
-			case (image : OpenGLUv) =>
-				this isAligned(image channels * image size x) ? this toRaster(image) : super(image)
-			case => super(source)
+		result: RasterImage
+		version(optiGraphicbufferupload) {
+			result = match (source) {
+				case (image : OpenGLMonochrome) =>
```

Comment:
Not your fault *this* time, but it got merged 7 months ago, so I'm off the hook as well. No colon before `:`.